metadata.json

```diff
@@ -1,23 +1,39 @@
 {
-  "name": "puppetlabs-distelli_agent",
-  "version": "0.1.0",
-  "author": "Puppet Inc",
-  "summary": "Installs, configures, and manages the Distelli agent service.",
+  "name": "puppetlabs-pipelines",
+  "version": "1.0.0",
+  "author": "Puppet, Inc.",
+  "summary": "Manages the Puppet Pipelines agent.",
   "license": "Apache-2.0",
-  "source": "https://github.com/puppetlabs/distelli-agent",
-  "project_page": "https://github.com/puppetlabs/distelli-agent",
+  "source": "https://github.com/puppetlabs/pipelines",
+  "project_page": "https://github.com/puppetlabs/pipelines",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
-    { "name":"puppetlabs/stdlib","version_requirement":">= 4.13.1 < 5.0.0" },
-    { "name":"puppet/archive","version_requirement":">= 2.0.0" }
+    {
+      "name": "puppet/archive",
+      "version_requirement": ">= 2.0.0 < 3.0.0"
+    },
+    {
+      "name": "puppetlabs/chocolatey",
```

Comment:
Might be missing something, but it looks like some of the chocolatey integration was removed... is this dependency still needed?

Reply:
@Iristyle Based on the fourth bullet at the top I think so.

Reply:
The module per se isn't dependent on it, but people may want to use chocolatey to manage things that the module is dependent on.

You can use the module without chocolatey, so I'll remove the dependency.